frontend/mission-components/app/globals.css

```
@@ -128,7 +128,7 @@
   /* Slate 600 */
 }
 
-::selection {
+::-selection {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: -selection

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)